chia/_tests/core/data_layer/test_data_store.py

```diff
@@ -255,7 +255,7 @@ async def test_build_a_tree(
     example = await create_example(data_store, tree_id)
 
     await _debug_dump(db=data_store.db_wrapper, description="final")
-    actual = await data_store.get_tree_as_program(tree_id=tree_id)
+    actual = await data_store.get_tree_as_nodes(tree_id=tree_id)
```

Comment:
In these tests I initially used `Program` ...  because...  seemed like it fit with our way.  And I guess early on I thought maybe we would treat the values as clvm etc.  But, we didn't go that route and nothing runtime actually gets this tree IIRC.  I decided to just remove this clvm dependency from DataLayer.  I think I could easily separate this into an isolated PR if you would like.

Reply:
No its fine as I read more of the PR it became more clear why you were doing it :+1: